docs/src/main/sphinx/connector/druid.rst

```diff
@@ -41,3 +41,11 @@ secured by basic authentication by updating the URL and adding credentials:
 
 Now you can access your Druid database in Trino with the ``druiddb`` catalog
 name from the properties file.
+
+.. _druid-sql-support:
+
+SQL support
+-----------
+
+The connector provides :ref:`read access <sql-read-operations>` to data and
+metadata in the Druid database.
```

Comment:
We still need to link to the globally available statements, otherwise it's not actually global. Suggest changing this to:

The connector provides :ref:`globally available
 <sql-globally-available>` and :ref:`read operation <sql-read-operations>`
 statements to access data and metadata in the Druid database.

Reply:
True.